Formula/cryfs.rb

```diff
@@ -0,0 +1,32 @@
+require "open3"
+
+class Cryfs < Formula
+  desc "Encrypts your files, so you can safely store them in Dropbox, iCloud, ..."
+  homepage "https://www.cryfs.org"
+  url "https://github.com/cryfs/cryfs.git", tag: "0.9.7"
+  head "https://github.com/cryfs/cryfs.git", branch: "develop"
+
+  depends_on "cmake" => :build
+  depends_on :python => :build
+  depends_on "openssl"
+  depends_on "boost"
+  depends_on "cryptopp"
+  depends_on :osxfuse
+
+  needs :cxx11
+
+  def install
+    mkdir("build") do
+      #TODO Homebrew is passing in CXXFLAGS to cmake which disable -O3. Can it be built with -O3?
+      system "cmake", "..", "-DBUILD_TESTING=off", *std_cmake_args
+      system "make", "install", "prefix=#{prefix}"
+    end
+  end
+
+  test do
+    Open3.popen3("#{bin}/cryfs") do |stdin, stdout, _|
```

Comment:
this is unfortunately not possible, since that would need calling `mount` and when I try doing that in a homebrew test, it tells me I don't have the permissions to call `mount`.